### PR TITLE
[Justice Counts] Help Center Bulk Upload copy changes

### PR DESCRIPTION
**Pull request by morden35**
## Description of the change

Filled in Help Center text for the following Bulk Upload sections:
- Single Page with Multiple Metrics
- Breakdown Column 
- Superagency Agency Column

## Type of change

> All pull requests must have at least one of the following labels applied (otherwise the PR will fail):

| Label                       	| Description                                                                                               	|
|-----------------------------	|-----------------------------------------------------------------------------------------------------------	|
| Type: Bug                   	| non-breaking change that fixes an issue                                                                   	|
| Type: Feature               	| non-breaking change that adds functionality                                                               	|
| Type: Breaking Change       	| fix or feature that would cause existing functionality to not work as expected                            	|
| Type: Non-breaking refactor 	| change addresses some tech debt item or prepares for a later change, but does not change functionality    	|
| Type: Configuration Change  	| adjusts configuration to achieve some end related to functionality, development, performance, or security 	|
| Type: Dependency Upgrade      | upgrades a project dependency - these changes are not included in release notes                             |

## Related issues

Closes #XXXX

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/HelpCenter/Guides/BulkUploadGuide.tsx [hunk 1]
```diff
@@ -137,25 +150,11 @@ export const BulkUploadGuide = () => (
     <Styled.SectionWrapper>
       <Styled.SectionTitle>Superagency Agency Column</Styled.SectionTitle>
       <Styled.SectionParagraph>
-        [Copy what we have in the Multiple Agencies section of the Bulk Upload
-        Instructions page]
-      </Styled.SectionParagraph>
-    </Styled.SectionWrapper>
-
-    <Styled.SectionWrapper>
-      <Styled.SectionTitle>Errors and Warnings</Styled.SectionTitle>
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/BulkUploadGuide.tsx:146]: moved this section below

File: publisher/src/components/HelpCenter/Guides/BulkUploadGuide.tsx [hunk 2]
```diff
@@ -21,10 +21,6 @@ import * as Styled from "../HelpCenter.styles";
 export const BulkUploadGuide = () => (
   <>
     <Styled.SectionWrapper>
-      <Styled.SectionParagraph>
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/BulkUploadGuide.tsx:24]: this was showing twice (same as caption)

**Review by mxosman — APPROVED** [review]
Looks great to me, Michelle! Thank you so much! I have just a few super small quick side-requests if you don't mind:

1. Can you delete L96-98 (Expected Column section) if that section is no longer needed?
2. Can you delete L163-166 and L203-205? (In order to remove the **[Video? ...]** text)
3. Can you check that the Loom videos linked in this section are visible to the public and not just Recidiviz folk?

File: publisher/src/components/HelpCenter/Guides/BulkUploadGuide.tsx (hunk 2 above)
Comment (mxosman) [reply publisher/src/components/HelpCenter/Guides/BulkUploadGuide.tsx:24]: Ah great catch!

**Comment by morden35** [comment]
> Looks great to me, Michelle! Thank you so much! I have just a few super small quick side-requests if you don't mind:
> 
> 1. Can you delete L96-98 (Expected Column section) if that section is no longer needed?
> 2. Can you delete L163-166 and L203-205? (In order to remove the **[Video? ...]** text)
> 3. Can you check that the Loom videos linked in this section are visible to the public and not just Recidiviz folk?

Thanks @mxosman!

1. I think 'Expected Columns' is suppose to be a header I think (one step above 'Year, Month, and Value Columns', 'Breakdown Column', etc. I can definitely delete it, unless there is some special formatting we want to use for different header 'levels'.
2. Done! great catch
3. Interesting, when I sign out of Loom, I am not able to view the videos on the Help Center page. Are we allowed to make Loom videos public? Alternatively, could download the videos as mp4s and embed them that way?
Screenshot 2023-10-16 at 4 35 17 PM

**Comment by mxosman** [comment]
> > Looks great to me, Michelle! Thank you so much! I have just a few super small quick side-requests if you don't mind:
> > 
> > 1. Can you delete L96-98 (Expected Column section) if that section is no longer needed?
> > 2. Can you delete L163-166 and L203-205? (In order to remove the **[Video? ...]** text)
> > 3. Can you check that the Loom videos linked in this section are visible to the public and not just Recidiviz folk?
> 
> Thanks @mxosman!
> 
> 1. I think 'Expected Columns' is suppose to be a header (one step above 'Year, Month, and Value Columns', 'Breakdown Column', etc. I can definitely delete it, unless there is some special formatting we want to use for different header 'levels'.

Ohhh - that makes sense. Thank you for the clarification! Yeah, I guess we don't quite have a nested heading structure (probably because this is the only one that uses it). I'll make a note in the playtesting doc and figure something out with that!

> 3. Interesting, when I sign out of Loom, I am not able to view the videos on the Help Center page. Are we allowed to make Loom videos public? Alternatively, could download the videos as mp4s and embed them that way?
>    Screenshot 2023-10-16 at 4 35 17 PM

Hmm - that's a good question about making them public. I imagine it's OK if there's no PII being revealed, but downloading them as MP4s might be a better idea. I'll make a note to download those videos and embed!

Thank you, Michelle!

**Pull request merged**